packages/keymaps/src/browser/keymaps-frontend-contribution.ts

```diff
@@ -81,4 +86,17 @@ export class KeymapsFrontendContribution extends AbstractViewContribution<Keybin
         });
     }
 
+    registerToolbarItems(toolbar: TabBarToolbarRegistry): void {
+        toolbar.registerItem({
+            id: KeymapsCommands.OPEN_KEYMAPS_JSON.id,
+            command: KeymapsCommands.OPEN_KEYMAPS_JSON.id,
+            tooltip: 'Open Keyboard Shortcuts in JSON'
+        });
+    }
+
+    protected isKeybindingWidget(widget: Widget | undefined = this.tryGetWidget()): boolean {
+        return widget instanceof KeybindingWidget && widget.id === KEYBINDING_WIDGET_ID
```

Comment:
I believe you can simply use `KeybindingWidget.ID` instead of `KEYBINDING_WIDGET_ID`.

Reply:
Oh that was my mistake... Will correct it